CalPatRec/src/AgnosticHelixFinder_module.cc

```diff
@@ -1689,7 +1739,10 @@ namespace mu2e {
       ComboHit hhit(*hit);
       hhit._hphi = _tcHits[i].helixPhi + _tcHits[i].helixPhiCorrection * 2 * M_PI;
       hseed._hhits.push_back(hhit);
+      eDepSum = eDepSum + hhit.energyDep();
+      nComboHits++;
     }
+    float eDepAvg = eDepSum/nComboHits;
```

Comment:
This calculation doesn't produce the true average, due to the variable number of straw hits in panel hits.  The mathematically correct average is:
size_t nstrawhits(0);
loop ...
eDepSum  += hhit.energyDep()*hhit.nStrawHits();
nstrawhits += hhit.nStrawHits();
end loop ...
eDepAvg = eDepSum/nstrawhits;